modules/navier_stokes/test/tests/finite_volume/ins/lid-driven/lid-driven.i

```diff
@@ -23,9 +23,11 @@ rho = 1
 [Variables]
   [vel_x]
     type = INSFVVelocityVariable
+    qp_calculations = true
```

Comment:
Is there a way to know if we need this or not? I mean we should provide a guidance for the users so they can decide. Or is this just to make sure the exodiff checks out?

Reply:
I think I went through and made sure that (common) qp-consumer objects mark requirements for qp-calculations (see https://github.com/idaholab/moose/pull/23233/files#r1099323717), so I believe these input file modifications can actually be removed. However, if someone has forgotten to mark a consumer as needing qp calculations, then this input file option could be used to accomplish the same purpose.

Reply:
I will think more about whether there is a fool-proof way to detect whether qp-computations are required in the library per @GiudGiud's suggestion (https://github.com/idaholab/moose/pull/23233#discussion_r1100863857), and if so, then I can remove the input file option totally

Reply:
Ok I was able to delete the `qp_computations` parameter